pkg/security/certificate_manager.go

```diff
@@ -196,24 +204,56 @@ func (cm *CertificateManager) RegisterSignalHandler(
 
 // RegisterExpirationCache registers a cache for client certificate expiration.
 // It is called during server startup.
-func (cm *CertificateManager) RegisterExpirationCache(cache *ClientCertExpirationCache) {
-	cm.clientCertExpirationCache = cache
+func (cm *CertificateManager) RegisterExpirationCaches(
+	ctx context.Context, stopper *stop.Stopper, parentMon *mon.BytesMonitor,
+) error {
+	m := mon.NewMonitorInheritWithLimit(
+		"client-expiration-caches", 0 /* limit */, parentMon, true, /* longLiving */
+	)
+	acc := m.MakeBoundAccount()
+	m.StartNoReserved(ctx, parentMon)
```

Comment:
Can a single account be used for multiple structs? Should there be two accounts (one for each cache)?